server.go

```diff
@@ -2,44 +2,28 @@ package dmsghttp
 
 import (
 	"net/http"
-	"time"
 
-	"github.com/SkycoinProject/dmsg/cipher"
-	"github.com/SkycoinProject/dmsg/disc"
+	"github.com/SkycoinProject/dmsg"
 )
 
 // Server holds relevant data for server to run properly
-// Data includes Public / Secret key pair that identifies the server.
-// There is also port on which server will listen.
-// Optional parameter is Discovery, if none is provided default one will be used.
-// Default dicovery URL is stored as dmsghttp.DefaultDiscoveryURL
+// Data includes parameters to instantiate a dmsgclient and a port on which server will listen.
 type Server struct {
-	PubKey    cipher.PubKey
```

Comment:
I don’t think the dmsghttp.Server type is needed. It’s not any more complicated just to use http.Server directly. Keeping this may confuse people as this is similarly named to http.Server but has a different interface.

Reply:
we also dont need the client then 
all we need is the transport